common/src/test/java/com/datastrato/gravitino/json/TestDTOJsonSerDe.java

```diff
@@ -189,11 +190,41 @@ public void testColumnDTOSerDe() throws Exception {
     Assertions.assertEquals(expectedJson, serJson);
     ColumnDTO deserColumn = JsonUtils.objectMapper().readValue(serJson, ColumnDTO.class);
     Assertions.assertEquals(column, deserColumn);
+    Assertions.assertEquals(Column.DEFAULT_VALUE_NOT_SET, column.defaultValue());
 
     // test default nullable
     String json = "{\"name\":\"column\",\"type\":\"byte\",\"comment\":\"comment\"}";
     ColumnDTO deColumn = JsonUtils.objectMapper().readValue(json, ColumnDTO.class);
     Assertions.assertTrue(deColumn.nullable());
+
+    // test specify column default value
+    column =
+        ColumnDTO.builder()
+            .withName(name)
+            .withDataType(Types.DateType.get())
+            .withComment(comment)
+            .withDefaultValue(
+                new LiteralDTO.Builder()
+                    .withDataType(Types.DateType.get())
+                    .withValue("2023-04-01")
+                    .build())
+            .build();
+    String actual = JsonUtils.objectMapper().writeValueAsString(column);
+    String expected =
+        "{\n"
+            + "  \"name\": \"column\",\n"
+            + "  \"type\": \"date\",\n"
+            + "  \"comment\": \"comment\",\n"
+            + "  \"nullable\": true,\n"
+            + "  \"autoIncrement\": false,\n"
+            + "  \"defaultValue\": {\n"
+            + "    \"type\": \"literal\",\n"
+            + "    \"dataType\": \"date\",\n"
+            + "    \"value\": \"2023-04-01\"\n"
+            + "  }\n"
+            + "}";
+    Assertions.assertEquals(
+        JsonUtils.objectMapper().readTree(expected), JsonUtils.objectMapper().readTree(actual));
```

Comment:
Can you please add a tet about null defaultValue.

Reply:
193 line is for testing null defaultValue